content/posts/2021-05-12-glimcap-part-fourteen.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "The Glimcap 14"
+date: 2021-05-12T08:00:00-04:00
+author: "Heiwa and RibenaBerri"
+---
+
+# Glimcap 14
```

Comment:
```suggestion
```
No need to put the title in here, the blog software automatically adds it.

Reply:
One change and it should be good.

Reply:
@clone1018 I think I fixed it